### ISSUE TITLE
Questions about the code paper

### ISSUE THREAD

**Issue by CNS-King** [tilman151/rul-adapt#62]
Thank you very much for the help of your code.The cnn._dann.ipynb in doc/examples is from which paper?  I hope to study this paper so as to better understand the code. Could you please tell me which paper's code this is about?

**Comment by tilman151** [comment]
The original method is by [Ganin et al](http://proceedings.mlr.press/v37/ganin15.pdf) and [Da Costa et al](https://arxiv.org/pdf/1907.07480) used it for RUL estimation for the first time. They used an LSTM instead of a CNN but that should not make any significant difference.

**Comment by CNS-King** [comment]
I am very sorry to bother you again. Now I reproduce the method of Laten align, no matter the bearing data or the engine data. I had very poor results.

I would like to ask you how many training epochs and loss function coefficients you set when replicating.


Thank you very much for your help and guidance during this period of time. I am a postgraduate student from Northeast University of China, and my research direction is RUL. I'd like to stay in touch more regularly.   To facilitate easier communication moving forward, would you mind if I added your instant messaging contact?   I'd be happy to use whichever platform you prefer, such as Telegram or WhatsApp.   Could you share your details with me?

**Comment by tilman151** [comment]
Sorry for the delay. We can use WhatsApp, but I'd prefer to not share my number here. My website (krokotsch.eu) has my e-mail address. Just send me a mail and I'll reply with my number.

**Comment by CNS-King** [comment]
> Sorry for the delay. We can use WhatsApp, but I'd prefer to not share my number here. My website (krokotsch.eu) has my e-mail address. Just send me a mail and I'll reply with my number.

Thank you very much for your answer. I have sent an email to your email address. I am looking forward to your reply.
